packages/editor/src/components/post-template/index.js

```diff
@@ -38,7 +33,7 @@ export function PostTemplate() {
 
 	const { editPost } = useDispatch( editorStore );
 
-	if ( ! isViewable || isEmpty( availableTemplates ) ) {
+	if ( ! isViewable || ! Object.entries( availableTemplates ?? {} ).length ) {
```

Comment:
`availableTemplates` could potentially be nullish, so we're falling back to an empty object before checking if it's empty. If not nullish, it is always an object.

Reply:
Have you considered using `Object.keys`? It creates a smaller array, so intuitively it should be less wasteful 🙂 

Instead of the `?? {}` suffix I'd consider writing `isViewable || ! availableTemplates || ! Object.keys( ... ).length`.

Reply:
Both are good suggestions, thanks 👍 Addressed in https://github.com/WordPress/gutenberg/pull/49799/commits/2f1bf4d882e5865818d3083068748b693f1a6101